src/main/kotlin/domains/Block.kt

```diff
@@ -1,5 +1,14 @@
 package domains
 
-abstract class Block(open val position: Position) {
+abstract class Block protected constructor(open val position: Position) {
```

Comment:
```suggestion
abstract class Block(open val position: Position) {
```

중요한 부분은 아니지만 추상 클래스는 상속 없이 인스턴스 생성이 불가능하니 protected를 생략하셔도 되지 않을까요~?

Reply:
네! 맞네요 감사합니다